src/style.css

```diff
@@ -0,0 +1,79 @@
+/* #region rotobo font import */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+/* #endregion */
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  padding: 10px 0;
+}
+
+.nav {
+  padding: 0;
+  margin: 0;
+  align-items: center;
+}
+
+.nav_list {
+  width: 100%;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  margin: 0;
+  padding: 0;
```

Comment:
if you want to remove default values like margin and padding use ul, body, a, li tags for that